apps/platform-app/src/app/[locale]/(profile)/_actions/queries/findProfileBySlug.ts

```diff
@@ -0,0 +1,17 @@
+'use server'
+
+import { createProfileModel } from '@/app/[locale]/(profile)/_models/profile.model'
+import { getProfileBySlug } from '@/backend/profile/profile.service'
+import { withSentry } from '@/utils/errHandling'
+import { type Profile } from '@prisma/client'
+
+export const findProfileBySlug = withSentry(async (slug: Profile['slug']) => {
```

Comment:
**nit:** Why not just `slug: string`?

Reply:
Ofc at the end Profile['slug'] is just a string but this way it's clear that it comes from Profile.